MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,575 @@
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.IsVisible = false;
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting watermark
+
+You can display the watermark in the editor by defining the display attribute or using the`GenerateDataFormItem` event.
+
+#### Using attribute
+
+You can show the watermark in the editor by setting the `Prompt` in display attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+    [Display(Prompt = "Enter middle name")]
+    public string MiddleName { get; set; }
+    
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+You can show the watermark in the editor by using the `PlaceholderText` property in `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Description")
+            {
+                e.DataFormItem.PlaceholderText = "Enter description";
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting watermark color
+
+You can display the color for the watermark in the editor by using `GenerateDataFormItem` event.
+
+You can change the color for the watermark in the editor by using the `PlaceholderColor` property in `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Description")
+            {
+                e.DataFormItem.PlaceholderText = "Enter description";
+                e.DataFormItem.PlaceholderColor = Colors.MistyRose;
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Padding
+
+You can create a space around a label and editor by using `Padding` property in the `DataFormItem`.
+
+Here, `FirstName` data field will be changed from the default position.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.Padding = new Thickness(10,0,10,0);
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Changing label name
+
+By default property name should be displayed as label. You can change the label text by using display attribute or using the `GenerateDataFormItem` event.
+
+#### Using attribute
+
+You can change the label text by setting the `Name` in display attribute.
```

Comment:
You can change the label text by setting the `Name` or `ShortName` in display attribute.

Reply:
content updated